internal/integration/index_view_test.go

```diff
@@ -568,16 +567,20 @@ func TestIndexView(t *testing.T) {
 			assert.True(mt, cmp.Equal(specs, expectedSpecs), "expected specifications to match: %v", cmp.Diff(specs, expectedSpecs))
 		})
 		mt.RunOpts("options passed to listIndexes", mtest.NewOptions().MinServerVersion("3.0"), func(mt *mtest.T) {
-			opts := options.ListIndexes().SetMaxTime(100 * time.Millisecond)
+			opts := options.ListIndexes().SetBatchSize(1)
```

Comment:
This test is just checking that options are being passed to the server correctly, since maxTime is no longer relevant this test was updated to use batch size.